doc/content/documentation/array-table-and-struct.md

```diff
@@ -1,9 +1,9 @@
 +++
-title = "Array, Table and Struct"
+title = "Array, Table, Struct and Set"
 weight = 9
 +++
 
-Phel has two mutable datastructures. An array is an indexed sequential datastructure and a table is an associative datastructure. While PHP has one single datastructure to represents arrays and hash tables, Phel splits them in two individual datastructures.
+Phel has three mutable datastructures. An array is an indexed sequential datastructure and a table is an associative datastructure. While PHP has one single datastructure to represents arrays and hash tables, Phel splits them in two individual datastructures.
```

Comment:
You should give a summary of all three data structures here. Currently you say that there are three, but you list only two :)

Reply:
Oops, yes. But actually I think I will stick with "two", since `set` doesn't seems to be mutable in any way in its current implementation. Indeed, set operations only create new sets, but doesn't update them.